apps/artwork/routes.coffee

```diff
@@ -70,6 +69,7 @@ bootstrap = ->
       extend res.locals.helpers, helpers
       bootstrap res.locals.sd, data
       res.locals.sd.PARAMS = req.params
+      res.locals.sd.INCLUDE_SAILTHRU = data.artwork.fair?
```

Comment:
Wondering if I should safeguard the presents of artwork like so `data.artwork?.fair?`